frontend/components/core/ReTableInfo.vue

```diff
@@ -47,150 +47,150 @@
                 </span>
-              </span>
-              <div v-if="visibleActions" class="table-info__actions">
-                <re-action-tooltip
-                  v-for="action in filterActions"
-                  :key="action.index"
-                  :tooltip="action.tooltip"
-                >
-                  <ReButton
-                    :title="action.title"
-                    class="table-info__actions__button button-icon"
-                    @click="onActionClicked(action.name, item)"
+                <div v-if="visibleActions" class="table-info__actions">
+                  <re-action-tooltip
+                    v-for="action in filterActions"
+                    :key="action.index"
+                    :tooltip="action.tooltip"
                   >
-                    <svgicon
-                      v-if="action.icon !== undefined"
-                      :name="action.icon"
-                      width="12"
-                      height="13"
-                    />
-                  </ReButton>
-                </re-action-tooltip>
-              </div>
-              <ReModal
-                :modal-custom="true"
-                :prevent-body-scroll="true"
-                modal-class="modal-secondary"
-                :modal-visible="showModal === item.name"
-                modal-position="modal-center"
-                @close-modal="$emit('close-modal')"
-              >
-                <div>
-                  <p class="modal__title">Delete confirmation</p>
-                  <p>
-                    You are about to delete: <strong>{{ item.name }}</strong
-                    >. This process cannot be undone.
-                  </p>
-                  <div class="modal-buttons">
                     <ReButton
-                      class="button-tertiary--small button-tertiary--outline"
-                      @click="$emit('close-modal')"
+                      :title="action.title"
+                      class="table-info__actions__button button-icon"
+                      @click="onActionClicked(action.name, item)"
                     >
-                      Cancel
+                      <svgicon
+                        v-if="action.icon !== undefined"
+                        :name="action.icon"
+                        width="12"
+                        height="13"
+                      />
                     </ReButton>
-                    <ReButton
-                      class="button-secondary--small"
-                      @click="onActionClicked('confirm-delete', item)"
-                    >
-                      Yes, delete
-                    </ReButton>
-                  </div>
+                  </re-action-tooltip>
                 </div>
-              </ReModal>
+                <ReModal
+                  :modal-custom="true"
+                  :prevent-body-scroll="true"
+                  modal-class="modal-secondary"
+                  :modal-visible="showModal === item.name"
```

Comment:
I've just realized that we don't set the `showModal` data property in any place. What's the logic about `showModal`?

Reply:
showModal is the id of the item whose modal is visible, I will rename it to visibleModalId